packages/app/package.json

```diff
@@ -101,7 +101,6 @@
         "lodash",
         "mobx",
         "nanoid",
-        "path",
```

Comment:
Something, somewhere, was causing this internalization to fail with an inscrutable error: https://app.circleci.com/pipelines/github/cypress-io/cypress/37390/workflows/48a17960-8622-45a4-a3bf-ff73f4e2b33d/jobs/1504577?invite=true#step-111-31

My hypothesis is that something that was hoisted by `npm/design-system/package.json` is no longer available, so this stopped working across the repo. I was unable to determine what that "something" was though, so I just removed this line from the `app`, `launchpad`, and `frontend-shared` `package.json`s.

This only had one side-effect of breaking `path` in the FE tests, which we only use in one place and was easily replaced by a 1 line function: 4357e8198bfeaf1f2ea3ff0879a735da81b50213